_posts/2016/2016-04-25-hackathon-2016.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "ePages Hackathon 2016"
+date: "2016-04-25 07:16:17"
+icon: code
+categories: events
+authors: ["Birgit"]
+---
+
+Another year, another ePages hackathon.
+Spring is a nice season for hackdays, isn't it?
+The world comes alive, the days are longer, and our developers have even more energy... for coding {% emoji wink %}!
+But what exactly did we do?
+
+## Ready, steady...
+
+{% image blog/blog-hack16-prep.jpg 50% right %} {% endimage %}
+
+This year's hackathon should focus on app development.
+Now that this was clear, we started to collect more detailed topics and built individual teams.
+The Scrum Masters addressed all manner of things associated with the hackathon organisation:
+Meeting rooms had to be booked, working spaces to be coordinated, snacks and beverages to be organised, as well as the final topics had to be agreed.
```

Comment:
as well as the final topics had to be agreed. -> as well as agreeing upon the final topics.